bluebutton-server-app/src/test/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/ExplanationOfBenefitResourceProviderIT.java

```
@@ -422,26 +422,30 @@ public void searchForEobsHasNoDupes() throws FHIRException {
 		IGenericClient fhirClient = ServerTestUtils.createFhirClient();
 
-		loadedRecords.stream().filter(r -> r instanceof Beneficiary).map(r -> (Beneficiary) r).forEach(beneficiary -> {
-			Bundle searchResults = fhirClient.search().forResource(ExplanationOfBenefit.class)
-					.where(ExplanationOfBenefit.PATIENT.hasId(TransformerUtils.buildPatientId(beneficiary)))
-					.returnBundle(Bundle.class).execute();
-			Assert.assertNotNull(searchResults);
-
-			/*
-			 * Verify that the returned Bundle doesn't have any resources with duplicate
-			 * IDs.
-			 */
-			Set<String> claimIds = new HashSet<>();
-			for (BundleEntryComponent searchResultEntry : searchResults.getEntry()) {
-				String resourceId = searchResultEntry.getResource().getId();
-				if (claimIds.contains(resourceId))
-					Assert.assertFalse(claimIds.contains(resourceId));
-				claimIds.add(resourceId);
-			}
-			if (searchResults.getTotal() > 0)
-				Assert.assertFalse(claimIds.isEmpty());
-		});
+		try {
+			loadedRecords.stream().filter(r -> r instanceof Beneficiary).map(r -> (Beneficiary) r).forEach(beneficiary -> {
+				Bundle searchResults = fhirClient.search().forResource(ExplanationOfBenefit.class)
+						.where(ExplanationOfBenefit.PATIENT.hasId(TransformerUtils.buildPatientId(beneficiary)))
+						.returnBundle(Bundle.class).execute();
+				Assert.assertNotNull(searchResults);
+
+				/*
+				 * Verify that the returned Bundle doesn't have any resources with duplicate
+				 * IDs.
+				 */
+				Set<String> claimIds = new HashSet<>();
+				for (BundleEntryComponent searchResultEntry : searchResults.getEntry()) {
+					String resourceId = searchResultEntry.getResource().getId();
+					if (claimIds.contains(resourceId))
+						Assert.assertFalse(claimIds.contains(resourceId));
+					claimIds.add(resourceId);
+				}
+				if (searchResults.getTotal() > 0)
+					Assert.assertFalse(claimIds.isEmpty());
+			});
+		} catch (Exception e) {
+			e.printStackTrace();
```

You should not swallow/ignore exceptions like this.